deployment/localup/localup.sh

```diff
@@ -191,11 +187,37 @@ print_work_dir() {
   done
 }
 
+##############
+# rebuild sp #
+##############
+function rebuild() {
+  bash ${workspace}/../../build.sh
+  mkdir -p ${workspace}/${SP_DEPLOY_DIR}
+  for ((i=0;i<${SP_NUM};i++));do
+    mkdir -p ${workspace}/${SP_DEPLOY_DIR}/sp${i}
+    cp -rf ${sp_bin} ${workspace}/${SP_DEPLOY_DIR}/sp${i}/${sp_bin_name}${i}
+  done
+}
+
+#############
+# reset sps #
+#############
+reset_sp() {
+  stop_sp
+  reset_db
+  reset_store
```

Comment:
What's the difference of `reset_db` and `reset_store`?

Reply:
how about `reset_sqldb` and `reset_piece_store`?

Reply:
fixed